PocketKit/Sources/PocketKit/Collections/CollectionViewController.swift

```diff
@@ -318,6 +294,134 @@ private extension CollectionViewController {
         }
     }
 
+    /// Determines the stories layout in the native collection view on iPad mode with regular horizontal size class (including split view). Number of columns for the grid layout depends on device orientation and the view's width length. If view's width is less than 800, show 2 col, otherwise show 2 col if it is in portrait and 3 col if it is in landscape mode.
+    /// - Parameters:
+    ///   - stories: list of stories associated with collection to display
+    ///   - width: width that the section occupies
+    ///   - margin: padding adding to the side of the section
+    /// - Returns: section layout for stories on iPad view and regular horizontal size class
+    private func collectionSectionForWideLayout(with stories: [CollectionStory], width: CGFloat, margin: CGFloat) -> NSCollectionLayoutSection {
+        let numberOfColumns: CGFloat = showThreeColumnLayout ? 3 : 2
```

Comment:
instead of having a `showThreeColumnLayout`, we could maybe have a `numberOfColumns` property that does the exact same calculations. Seems more explicit and avoids one boolean property (which tend to be confusing as they might only represent part of the states that a view can have)

Reply:
update to 
```
    /// Determines whether to show a 3 or 2 column layout
    /// - Parameter view: view that we want to provide a layout for
    /// - Returns: true if width meets the minimum requirement and is not a portrait view
    private func numberOfColumns(with view: UIView) -> Int {
        let hasRequirementForThreeColumnLayout = view.bounds.width >= Constants.minWidthBoundaryForFullColumnLayout && !UIDevice.current.orientation.isPortrait
        return hasRequirementForThreeColumnLayout ? 3 : 2
    }
```